### ISSUE TITLE
No module named 'spikingjelly.activation_based'

### ISSUE THREAD

**Issue by esther95xin** [fangwei123456/spikingjelly#238]
I have removed spikingjelly installed before,(0.0.0.0.12) and reinstall through pip. However, I cannot load module in newest version.

**Comment by fangwei123456** [comment]
Hi, the latest version has changed the packages name. Refer to this:

https://spikingjelly.readthedocs.io/zh_CN/latest/activation_based_en/migrate_from_legency.html

**Comment by fangwei123456** [comment]
But we have not update the pip version. The pip version is still the old version.

![image](https://user-images.githubusercontent.com/38352109/174522370-b9698564-fe8e-4974-a1ea-b0eab6d7d5e8.png)

To use the latest developing version, you need to install from github source codes.

**Comment by fangwei123456** [comment]
If you use the old version, you can refer to the old version of docs:

https://spikingjelly.readthedocs.io/zh_CN/0.0.0.0.12/

**Comment by Atlantisming** [comment]
I met the same question and I found that there is a "spikingjelly" dir outside.
So the module named 'spikingjelly.activation_based' is in "spikingjelly.spikingjelly.activation_based"
you can try this code.
"""
from spikingjelly.spikingjelly.activation_based import functional, surrogate, neuron
from spikingjelly.spikingjelly.activation_based.model import parametric_lif_net
from spikingjelly.spikingjelly.datasets.dvs128_gesture import DVS128Gesture
"""

**Comment by fangwei123456** [comment]
@Atlantisming You can cd the dir and install it again:

```
git clone https://github.com/fangwei123456/spikingjelly.git
cd spikingjelly
python setup.py install
```

**Comment by bia1o** [comment]
I run spikingjelly/spikingjelly/activation_based/examples/lif_fc_mnist py when met the from spikingjelly. Activation_based import neuron, encoding, functional, surrogate, layer
ModuleNotFoundError: No module named 'spikingjelly. Activation_based ', I still failed to download the source code through Github again. What is the problem

**Comment by bia1o** [comment]
I checked my Project Interpreter and saw that spikingjelly Package was still 0.0.0.012

**Comment by fangwei123456** [comment]
You can try to delete the existed `spikingjelly` dir, uninstall `spikingjelly` by pip, re-download from github and re-install it.

**Comment by bia1o** [comment]
I am using a anaconda configuration environment, perform the git clone https://github.com/fangwei123456/spikingjelly.git

cd spikingjelly

Where is spikingjelly0.0.13 downloaded from python setup.py? Can I specify to download to my configured environment?

**Comment by fangwei123456** [comment]
>Where is spikingjelly0.0.13 downloaded

The current master branch in github is the 0013 version.

>Can I specify to download to my configured environment?

you can use the python in your env to achieve it. For example:

```
(base) wfang@mlg-ThinkStation-P920:~$ conda activate sj-dev
(sj-dev) wfang@mlg-ThinkStation-P920:~$ python setup.py install
```

**Comment by bia1o** [comment]
Is spikingJelly code still available for version 0.0.0.0.12? The current code is all based on 0.0.0.0.13？

**Comment by fangwei123456** [comment]
Of course. The version in pypi is 0012, and this version is also available in github:

https://github.com/fangwei123456/spikingjelly/tree/0.0.0.0.12

**Comment by bia1o** [comment]
Thank you very much. When will pYPI update version 0.0.0.0.13

**Comment by fangwei123456** [comment]
It will come in months and the you can install the version 000014 from pypi.

> The odd version number is the developing version, which is updated with GitHub/OpenI repository. The even version number is the stable version and available at PyPI.

**Comment by Ahmed1Zaky** [comment]
Hi, 

I have setup-the latest GitHub version of the package and I am still getting the same error: "No module named 'spikingjelly.activation_based'", have you changed the names again? What am I missing here?

**Comment by fangwei123456** [comment]
Hi, do you solve this problem now?

**Comment by ZHAODONG-LYU** [comment]
> Hi, do you solve this problem now?

no
